app/assets/v2/js/pages/profile-trust.js

```diff
@@ -648,6 +663,113 @@ Vue.component('brightid-verify-modal', {
+                      <p>
+                        Duniter is a free software for free currencies: a P2P, <a href="https://en.wikipedia.org/wiki/Web_of_trust">Web of Trust</a> & <a href="https://en.wikipedia.org/wiki/Social_credit">Universal Dividend system.</a>
+                        <p class="mb-4">
+                          <a href="https://duniter.org/en/introduction/" target=_blank>Learn more.</a>
+                        </p>
+                      </p>
+                      <div>
+                        <h2 class="font-bigger-4 font-weight-bold"> You need to have these requirements:</h2>
+                        <ol>
+                          <li>Let's check if there is already a record with your account.</li>
+                          <li>Link to your gitcoin account in your Duniter record </li>
+                          <li>If you are a qualified Duniter member</li>
+                        </ol>
+                      </div>
+                    </div>
+                    <div v-if="validationStep === 'validate-duniter' || validationStep == 'perform-validation'">
+                      <p class="mb-4">
+                        You fulfill all the requirements, if you just need to click on validate to confirm that your duniter account is valid
+                      </p>
+                      <b-button @click="clickedValidate" :disabled="validationStep === 'validate-duniter'" class="btn-gc-blue mt-3 mb-2" size="lg">
+                        <b-spinner v-if="validationStep === 'validate-duniter'" type="grow"></b-spinner>
+                        Validate
+                      </b-button>
+                      <div v-if="validationError !== ''" style="color: red">
+                        <small>[[validationError]]</small>
+                      </div>
+                      <br />
+                      <a href="" v-if="validationError !== ''" @click="clickedGoBack">
+                        Go Back
+                      </a>
+                    </div>
+                    <div v-if="validationStep === 'validation-complete'">
+                      Your Duniter verification was successful. Thank you for helping make Gitcoin more sybil resistant!
+                      <a href="" class="btn btn-gc-blue px-5 mt-3 mb-2 mx-2" role="button" style="font-size: 1.3em">Done</a>
+                    </div>
+                  </div>
+                </template>
+            </b-modal>`,
+  methods: {
+    dismissVerification() {
+      this.showValidation = false;
+    },
+    clickedGoBack(event) {
+      event.preventDefault();
+      this.validationStep = 'validate-duniter';
+      this.validationError = '';
+    },
+    clickedValidate(event) {
+      event.preventDefault();
+
+      this.validationError = '';
+
+      this.validationStep = 'perform-validation';
+      this.getUserHandle();
+      this.verifyDuniter();
+    },
+    getUserHandle() {
+      this.githubHandle = trustHandle;
```

Comment:
I don't see where `trustHandle` is defined, is this code working? I suppose is a global variable defined out of vue context, if that the case, ok but if not then this will be undefined.

Reply:
@developerfred